### ISSUE TITLE
Toolbox magic type title

### ISSUE THREAD

**Issue by suraj740** [apache/incubator-echarts#12979]
### Version
4.8.0

### Reproduction link
[https://jsfiddle.net/guptasuraj/sq64rko3/4/](https://jsfiddle.net/guptasuraj/sq64rko3/4/)

### Steps to reproduce
On first load magicType feature title would be in english, but after clicking on stack icon it resets the language to chinese.

### What is expected?
language of the tile should be what we have provided in title object  as below:

`title: {
                  'line': 'Line chart',
                  'bar': 'Bar chart',
                  'stack': 'Stack'
              }`

### What is actually happening?
But its changes to chinese language after clicking on stack icon.

**Comment by plainheart** [comment]
Thanks for reporting. This issue is similar to #12667. It may be fixed in 4.9.0.

**Comment by pingf** [comment]
right now(4.9), I can rename the icon title of magicType in this way
```js
magicType: {
          show: true,
          type: ['line', 'bar', 'stack'],
          title: {
            line: 'line view',
            bar: 'bar view',
            stack: 'stack view',
          },
        },
```

**Comment by suraj740** [comment]
@pingf once you toggle stack view, it changes back to chinese language.

**Comment by pingf** [comment]
@suraj740 
arrr... I see.
I think I can create a PR to fix this problem, cause stack button is a button with states, 
right now, it use the default title to overwrite when it changes

**Comment by pingf** [comment]
@suraj740 
I've created a PR, hoping it can be merged soon
one more thing is that, the title for stack and tiled are in pair(we dont need to set the tiled in type)
so the option for title maybe sth like

```js
magicType: {
                    show: true,
                    type: ['line', 'bar', 'stack'],
                    title: {
                        line: 'line view',
                        bar: 'bar view',
                        stack: 'stack view',
                        tiled: 'tiled view',
                    },
                },
```

**Comment by suraj740** [comment]
@pingf  Great, Thanks!!

**Comment by asabi** [comment]
Is this out already? I am still getting the Chinese version after clicking the toggle.

**Comment by drumslave-git** [comment]
Yep, it is still actual in 4.9.0

**Comment by pissang** [comment]
It should have been fixed since 5.0

**Comment by pingf** [comment]
yep, the pr is merged to 5.0's branch
